dev/cognitiveservices/data-plane/Language/analyzetext-authoring.json

```diff
@@ -2782,6 +3036,59 @@
       "discriminator": "projectKind",
       "x-ms-client-name": "ExportedProjectAssets"
     },
+    "TextAnalysisAuthoringExportedTA4HDocument": {
+      "description": "Represents an exported document for a TA4H project.",
+      "type": "object",
+      "properties": {
+        "entities": {
+          "description": "The list of entity labels belonging to the document.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TextAnalysisAuthoringExportedDocumentRegion"
```

Comment:
Brought up in the meeting, but I'll mention it here for ease: even though this is an old model, since we're rev'ing the version, `TextAnalysisAuthoringExportedDocumentRegion` should also declare a `StringIndexType` like other region-like models that specify an offset. It may still be hardcoded like some others, but should be explicit so clients / callers know how to handle different source encodings e.g.,  UTF8 JSON -> UTF16 strings in some languages like JavaScript or .NET.